mantidimaging/gui/windows/operations/view.py

```diff
@@ -280,3 +289,15 @@ def toggle_filters_section(self):
         else:
             self.splitter.setSizes([200, 9999])
             self.collapseToggleButton.setText("<<")
+
+    def set_initial_filter(self, filter_arg: str):
+        """
+        Sets the initial filter in the operations window.
+        :param filter_arg: The name of the filter given by the user.
+        """
+        filter_arg = filter_arg.replace("-", "").lower()
+        for i in range(self.filterSelector.count()):
+            filter_text = self.filterSelector.itemText(i)
+            if _strip_filter_name(filter_text) == filter_arg:
+                self.filterSelector.setCurrentIndex(i)
+                return
```

Comment:
Could this use `FiltersWindowPresenter.set_filter_by_name()`, maybe can reduce duplication with `MainWindowPresenter.wizard_action_show_operation()`.